ui/app/models/aws/root-config.js

```diff
@@ -45,40 +45,50 @@ export default class AwsRootConfig extends Model {
   iamEndpoint;
   @attr('string', { label: 'STS endpoint' }) stsEndpoint;
   @attr('number', {
-    label: 'Maximum retries',
```

Comment:
So here's the deal: calling this 'Maximum' when the api attr is `Max` makes things messy when running through tests. I'm doing and asking for permission later here.